src/basecases.py

```diff
@@ -158,7 +218,7 @@ def bootstrap_random_dag(type_num=4, complexity="low", nodes_per_type=2):
             demo_nodes.append(node_tmp)
 
         else:
-            for j in range(1, random.randint(2, nodes_per_type)):
```

Comment:
Actually keep this part for now. We're short on time, so it's okay to have this in,

Reply:
Add random.seed() in random_dag(nodes, edges) to make the dag creation somewhat fixed.

Reply:
DONE!